### DISCUSSION TITLE
Metal Gear Solid (v1.0 & 1.1) Seemingly Incompat w/ Adam Img?

### DISCUSSION THREAD

**Discussion by Beezleybuzz** [eduardofilo/RG350_adam_image#90] [General]
Have been tinkering with the img on my RG350M, testing games and such. Ran across potential weird issue with Metal Gear Solid. Will not boot. Have tried multiple different PSX bios (SPCH-7001, 7502, 5501) without any luck. Reverting to stock SD/img results in game booting no issue. 

Any thoughts or suggestions?

**Comment by eduardofilo** [comment]
I have been trying and it works to me in CHD format.

I adjusted the emulator to show the machine's boot logos to detect problems with the BIOS. If you don't see the logos, then the problem is in the BIOS. You have to put the version indicated in the [README table](https://github.com/eduardofilo/RG350_adam_image#bios). Of course you can also use another BIOS, but I can only ensure that it works with the one I put there.
If you see the logos, then the problem is with the game. Search for another version.

**Comment by lagwagone** [comment]
Just a note on this - for MGS to work you need to set HLE Emulated BIOS to off in PSX4ALL Core Settings
